watertap/property_models/unit_specific/activated_sludge/tests/test_modified_asm2d_thermo.py

```diff
@@ -204,13 +264,13 @@ def test_build(self, model):
         metadata = model.params.get_metadata().properties
 
         # check that properties are not built if not demanded
-        for v in metadata.list_supported_properties():
-            if metadata[v.name].method is not None:
-                if model.props[1].is_property_constructed(v.name):
-                    raise PropertyAttributeError(
-                        "Property {v_name} is an on-demand property, but was found "
-                        "on the stateblock without being demanded".format(v_name=v.name)
-                    )
+        # for v in metadata.list_supported_properties():
+        #     if metadata[v.name].method is not None:
+        #         if model.props[1].is_property_constructed(v.name):
+        #             raise PropertyAttributeError(
+        #                 "Property {v_name} is an on-demand property, but was found "
+        #                 "on the stateblock without being demanded".format(v_name=v.name)
+        #             )
```

Comment:
Why comment these out?

Reply:
I think this test was written to check that the constraints for VSS, ISS, TSS, COD, etc. were only built on-demand, but I've changed them from constraints to expressions, so I don't think this test applies anymore. On that note, I don't think I need to be touching the effluent properties in the flowsheet anymore either.